### PR TITLE
refactor get_token_from_header

### PR DESCRIPTION
**Pull request by SimonLab**
Update the `get_token_from_header` to avoid broken if/else statement
ref:  #32

### CODE REVIEW

File: lib/token.ex [hunk 1]
```diff
@@ -137,7 +137,7 @@ defmodule AuthPlug.Token do
       # Check for JWT in Headers:
       Enum.count(get_req_header(conn, "authorization")) > 0 ->
         conn.req_headers
-        |> List.keyfind("authorization", 0)
+        |> List.keyfind("authorization", 0, "")
```
Comment (SimonLab) [inline lib/token.ex:140]: Return empty string if "authorization" not find in the list

Comment (nelsonic) [reply lib/token.ex:140]: Makes sense to have a default. ✅

File: lib/token.ex [hunk 2]
```diff
@@ -202,24 +202,19 @@ defmodule AuthPlug.Token do
     AuthPlug.Token.create_session(conn, claims, jwt)
   end
 
-  #  fail fast if no JWT in auth header:
-  defp get_token_from_header(nil), do: nil
```
[the remark this answers is not in the document]
Comment (nelsonic) [reply lib/token.ex:206]: Makes sense. Thanks for deleting. ✂️

**Comment by nelsonic** [comment]
Package published to https://hex.pm/packages/auth_plug/1.3.2 📦 :shipit: 
(thanks again @SimonLab)

**Pull request merged**